pkg/packetbrokeragent/grpc_pba.go

```diff
@@ -39,10 +39,15 @@ type pbaServer struct {
 	cpConn *grpc.ClientConn
 }
 
+var errNotEnabled = errors.DefineFailedPrecondition("not_enabled", "Packet Broker is not enabled")
```

Comment:
A noop `pbaServer` would have been cleaner I think, but this is fine as is.

Reply:
Yeah I agree, I'll keep it in mind for future PBA work.